chatminer/chatparsers.py

```diff
@@ -230,19 +231,19 @@ def _parse_message(self, mess):
         elif any(key == "is_unsent" for key in mess):
             return None
         else:
-            self._logger.warning("Skipped message with unknown format: %s", mess)
+            self._logger.warning(f"Skipped message with unknown format: {mess}")
             return None
 
-        time = datetime.datetime.fromtimestamp(mess["timestamp_ms"] / 1000)
+        time = dt.datetime.fromtimestamp(int(mess["timestamp_ms"]) / 1000)
```

Comment:
Can we assume that `timestamp_ms` is not a float? Doesn't make a difference in the end, but it's not clear to me why we would need an explicit typecast here

Reply:
I'm not 100% sure why I did this but I think it was because I defined mess as type Dict[str, str] while it should actually be Dict[str, Any]. I will fix this in my next commit.